sectionsUtilityTool/src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionUtilityToolFilter.java

```diff
@@ -0,0 +1,171 @@
+package edu.umich.ctools.sectionsUtilityTool;
+
+import java.io.IOException;
+import java.util.Hashtable;
+import java.util.Properties;
+
+import javax.naming.Context;
+import javax.naming.NamingEnumeration;
+import javax.naming.NamingException;
+import javax.naming.directory.Attribute;
+import javax.naming.directory.DirContext;
+import javax.naming.directory.InitialDirContext;
+import javax.naming.directory.SearchControls;
+import javax.naming.directory.SearchResult;
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import edu.umich.its.lti.utils.PropertiesUtilities;
+
+
+
+public class SectionUtilityToolFilter implements Filter {
```

Comment:
If the value can be read from a properties file then testing can be done
with the production LDAP but doesn't need to be.  Why require a new build
later if that is trivial to make the code more flexible now?

- Dave

On Mon, Mar 2, 2015 at 2:29 PM, Pushyami Gundala <notifications@github.com>
wrote:

> In
> sectionsUtilityTool/src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionUtilityToolFilter.java
> <https://github.com/tl-its-umich-edu/canvas/pull/16#discussion_r25629327>:
>
> > +import javax.servlet.FilterChain;
> > +import javax.servlet.FilterConfig;
> > +import javax.servlet.ServletException;
> > +import javax.servlet.ServletRequest;
> > +import javax.servlet.ServletResponse;
> > +import javax.servlet.http.HttpServletRequest;
> > +import javax.servlet.http.HttpServletResponse;
> > +
> > +import org.apache.commons.logging.Log;
> > +import org.apache.commons.logging.LogFactory;
> > +
> > +import edu.umich.its.lti.utils.PropertiesUtilities;
> > +
> > +
> > +
> > +public class SectionUtilityToolFilter implements Filter {
>
> We will be using prod ldap server for local testing as well
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tl-its-umich-edu/canvas/pull/16/files#r25629327>.
>